Formula/t/technitium-dns.rb

```diff
@@ -0,0 +1,68 @@
+class TechnitiumDns < Formula
+  desc "Self host a DNS server for privacy & security"
+  homepage "https://technitium.com/dns/"
+  url "https://github.com/TechnitiumSoftware/DnsServer/archive/refs/tags/v13.5.0.tar.gz"
+  sha256 "7a5f47af4c938c5983e415547b777f853ede5ada8837012c8b5e8e36df1a380e"
+  license "GPL-3.0-or-later"
+
+  depends_on "dotnet@8"
+  depends_on "libmsquic"
+
+  on_linux do
+    depends_on "bind" => :test # for `dig`
+  end
+
+  resource "TechnitiumLibrary" do
+    url "https://github.com/TechnitiumSoftware/TechnitiumLibrary/archive/refs/tags/dns-server-v13.5.0.tar.gz"
+    sha256 "8e0f3200ad6ee6a1ab346a7224a1bc538f6b3dc1b660cd8bb725d0dc09fa2bf9"
+  end
+
+  def install
+    # See https://github.com/TechnitiumSoftware/DnsServer/blob/master/build.md#build-instructions
+    dotnet = Formula["dotnet@8"]
+    mkdir_p "DnsServer"
+    mv Dir.glob("*").reject { |f| f == "DnsServer" }, "DnsServer"
+    resource("TechnitiumLibrary").stage do
+      system dotnet.opt_bin/"dotnet", "build",
+             "TechnitiumLibrary.ByteTree/TechnitiumLibrary.ByteTree.csproj",
+             "-c", "Release"
+      system dotnet.opt_bin/"dotnet", "build",
+             "TechnitiumLibrary.Net/TechnitiumLibrary.Net.csproj",
+             "-c", "Release"
+      cp_r ".", buildpath/"TechnitiumLibrary"
+    end
+    system dotnet.opt_bin/"dotnet", "publish", "DnsServer/DnsServerApp/DnsServerApp.csproj", "-c", "Release"
+    libexec.install Dir["DnsServer/DnsServerApp/bin/Release/publish/*"]
+  end
+
+  service do
+    dotnet = Formula["dotnet@8"]
+    run [dotnet.opt_bin/"dotnet", opt_libexec/"DnsServerApp.dll", etc/"technitium-dns"]
```

Comment:
No need to define it if you only use it once. It might be easier to make a wrapper script though.
```suggestion
    run [Formula["dotnet@8"].opt_bin/"dotnet", opt_libexec/"DnsServerApp.dll", etc/"technitium-dns"]
```

Reply:
Do you happen to have a good example formula for a wrapper script? I'd move the `DYLD_FALLBACK_LIBRARY_PATH` there. It might also be a good place to delay startup until the network is reachable (e.g. during system boot).